TenancyInformationApi.Tests/V1/UseCase/ListTenanciesTests.cs

```diff
@@ -74,8 +74,8 @@ public void IfLimitLessThanTheMinimumWillUseTheMinimumLimit()
         [Test]
         public void IfLimitMoreThanTheMaximumWillUseTheMaximumLimit()
         {
-            SetupMockGatewayToExpectParameters(limit: 100);
-            CallUseCaseWithArgs(400, 0);
+            SetupMockGatewayToExpectParameters(limit: 1000);
+            CallUseCaseWithArgs(4736, 0);
```

Comment:
Was this number selected deliberately? Does it refer to anything? It looks really specific 😅

Reply:
just random, but over 1000.